doc/usage/quickstart.rst

```diff
@@ -308,7 +324,3 @@ More topics to be covered
 .. [#] This is the usual layout.  However, :file:`conf.py` can also live in
        another directory, the :term:`configuration directory`.  Refer to the
        :doc:`sphinx-build man page </man/sphinx-build>` for more information.
-
-.. |more| image:: /_static/more.png
-          :align: middle
-          :alt: more info
```

Comment:
This code removal puzzled me for a few moments; now I understand it's removal of a substitution definition that is no-longer-used on the page (a bit like an unused import).